File: cpp/Mahasiswa.cpp
class Mahasiswa
{
	private:
		string nama;
		string nim;
		string prodi;
		string fakultas;
	public:
		Mahasiswa(){
		}
		Mahasiswa(string nama,string nim, string prodi, string fakultas){
			this->nama = nama;
			this->nim = nim;
			this->prodi = prodi;
			this->fakultas = fakultas;
		}
		void setNama(string nama){
			this->nama = nama;
		}
		string getNama(){
			return this->nama;
		}
		void setNim(string nim){
			this->nim = nim;
		}
		string getNim(){
			return this->nim;
		}
		void setProdi(string prodi){
			this->prodi = prodi;
		}
		string getProdi(){
			return this->prodi;
		}
		void setFakultas(string fakultas){
			this->fakultas = fakultas;
		}
		string getFakultas(){
			return this->fakultas;
		}
};

File: python/Mahasiswa.py
class Mahasiswa:
    def __init__(self, nama, nim, prodi, fakultas):
        self.nama = nama
        self.nim = nim
        self.prodi = prodi
        self.fakultas = fakultas
    
    def setNama(self, nama):
        self.nama = nama

    def getNama(self):
        return self.nama

    def setNim(self, nim):
        self.nim = nim

    def getNim(self):
        return self.nim
	
    def setProdi(self, prodi):
        self.prodi = prodi
	
    def getProdi(self):
        return self.prodi
	
    def setFakultas(self, fakultas):
        self.fakultas = fakultas
	
    def getFakultas(self):
        return self.fakultas
	